website/src/blogs/how-to-install-ibm-cloud-private-updated.md

```diff
@@ -0,0 +1,281 @@
+
+    TASK [addon : Waiting for cloudant to start] ******************************************
+     *********************************************************************
+    FAILED — RETRYING: TASK: addon :
+    Waiting for cloudant to start (13 retries left).
+    FAILED — RETRYING: TASK: addon :
+    Waiting for cloudant to start
+
+Double check your h/w resources and run the installer with verbose options to see more details:
+
+    docker run -e LICENSE=accept — net=host \
+    -t -v “$(pwd)”:/installer/cluster \
+    ibmcom/icp-inception:3.1.0 install -vvv | tee -a install_log.txt
+
+I ended up increasing memory from 8GB to 16GB and disk capacity to 200GB (from 150GB) and all worked well after that.
+
+After a successful install you should see a message similar to the below:
+
+    PLAY RECAP *********************************************************************
+     10.10.0.161 : ok=173 changed=94 unreachable=0 failed=0
+     10.10.0.162 : ok=113 changed=55 unreachable=0 failed=0
+     10.10.0.163 : ok=108 changed=51 unreachable=0 failed=0
+     10.10.0.164 : ok=108 changed=50 unreachable=0 failed=0
+     10.10.0.165 : ok=108 changed=49 unreachable=0 failed=0
+     localhost : ok=265 changed=161 unreachable=0 failed=0POST DEPLOY MESSAGE ************************************************************The Dashboard URL: https://10.10.0.161:8443, default username/password is admin/admin
+    Playbook run took 0 days, 0 hours, 53 minutes, 54 seconds
+
+One thing I have noticed is that the installation completed much faster than the previous version. ICP got smarter and only pulling images that are required for the roles, also seems like taking advantage of the local image registries. My 5 node installation time dropped from 95 minutes to 53 minutes. If your deployment is successful, you should be able to access your ICP login screen by visiting `https://MASTERNODEIP:8443` (Default username/password is admin/admin).
+![https://cdn-images-1.medium.com/max/800/0icEkTNw9gTeerOUW.png](https://lh4.googleusercontent.com/zLjFhjjYasubn8qOstAt53Dk-KONjCxMXqngIhhYqLyRWKt3C4UOJI_hc4IBcZ7KA6qefUGP1GQIUDF14QQZuKSYM64g5sDbJgspI4dAtzTRQOVE_-71xUNh6wgfcZwiBEmLFgSCJF8A7XJHeA)IBM Cloud Private Login Screen![https://cdn-images-1.medium.com/max/800/035UO58nb7V2f6Lwr.png](https://lh4.googleusercontent.com/gg0FToYgEP-8WMOZxm80wyHDFPx_cXO8VMrEIhM-YXIR_jWQiUQuWU12Ttj1iWALZv3O60yYcZWjwg0TKuh-4g_m7Jdf2JbhmOQEEHHvSiau7H492Da5AZF6iMeh7AmXy0NONQeTJgrkpaaz1w)IBM Cloud Private Dashboard![https://cdn-images-1.medium.com/max/800/0xtFeNgX5Z-iPQQFS.png](https://lh6.googleusercontent.com/vTnBDyMf3S-Rct0ideYAJTMMAiUqChItSPxRghrtbFLtf7koXrGuPPcZjFAYvEdD6kVu0gAuAHYRBy1xHnRWADOOqQwtRJN4sgcMMCWFJqJwxjYEh5Y53xgOcRQ_INjZ2bK61cLHVF3aD6Wlrw)IBM Cloud Private Catalog
```

Comment:
Please fix these images as per our meet discussion. I think we wanted to use `medium link` images as images. cc @satyapriyamishra222